src/Router.tsx

```diff
@@ -70,7 +70,7 @@ function AttemptLogin(): React.ReactElement {
     return <Navigate to="/login" />;
   }
 
-  return <Navigate to="/" />;
+  return <Navigate to="/" replace />;
```

Comment:
I would *replace" the browser history also in the redirection above, i.e. when navigating to Login in case of unsuccessful login.
Otherwise the user won't be able to go back to the previously visited website.

Same thing for the Login and Logout components below, that could *replace* redirect to `/logout` and `/login` respectively.